libflux/flux/src/bin/flux-dump-docs.rs

```diff
@@ -0,0 +1,6 @@
+use flux::docs;
+
+fn main() {
+    let doc = docs();
```

Comment:
I was expecting this to call a `doc_json` function that returns the raw json so that we do not have to use `serde` to serialize the docs again. Am I reading this code wrong?

Reply:
No you're right, I ended up scrapping the doc_json function and implementing the conversion the same way we do it in wasm's mod.rs for consistency. I could change it and add the function back, but I thought this would be easier to implement.